internal/mode/static/state/graph/secret_test.go

```diff
@@ -186,15 +186,17 @@ func TestSecretResolver(t *testing.T) {
 	// Not running tests with t.Run(...) because the last one (getResolvedSecrets) depends on the execution of
```

Comment:
Shoot, I did not see this comment before. Let's revert this to not using t.Run

Reply:
Thanks for catching that, besides that, was it safe for me to add subtests to all the other tests?

Reply:
Should be, unless there are any similar comments.